modules/openapi-generator/src/main/resources/typescript/api_doc.mustache

```diff
@@ -22,27 +22,25 @@ Method | HTTP request | Description
 
 
 ```typescript
-import { {{{moduleName}}} } from '{{{projectName}}}';
-import * as fs from 'fs';
+import {{{moduleName}}} from '{{{projectName}}}';
```

Comment:
we're using a default export here, alternatively we could do:
```suggestion
import { {{classname}}, createConfiguration } from '{{{projectName}}}';
```
if we want to keep the non-default export.